app/actions/sessions.js

```diff
@@ -151,6 +154,67 @@ export function getScheduledQueries() {
     };
 }
 
+export function createScheduledQuery(connectionId, payload = {}) {
+  return (dispatch) => {
+    return dispatch(apiThunk(
+      'queries',
+      'POST',
+      'createScheduledQueryRequest',
+      payload.filename,
+      {
+        requestor: payload.requestor,
+        uids: payload.uids,
+        fid: payload.fid,
+        filename: payload.filename,
+        refreshInterval: payload.refreshInterval,
+        query: payload.query,
+        connectionId
+      }
+    )).then((res) => {
+      dispatch(createScheduledQueryAction(res));
+      return res;
+    });
+  };
+}
+
+export function updateScheduledQuery(connectionId, payload = {}) {
```

Comment:
See [this comment](https://github.com/plotly/falcon-sql-client/pull/476/files#r200085883).